### DISCUSSION TITLE
Welcome to rails-world Discussions!

### DISCUSSION THREAD

**Discussion by JGTR** [TelosLabs/rails-world#83] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yousamaa** [comment]
Hi I am Usama. I have over 2 years of experience with Ruby On Rails and I want to make contributions to this app. How can I do that. Please guide me because I am new to open source. I am also attending Rails world this year if my visa application goes well. Thanks.

**Comment by yousamaa** [comment]
@JGTR ☝️

**Comment by JGTR** [comment]
Hi @yousamaa Thanks for posting! @LuigiR0jas will reply with a few items that you could definitely help us out with. Good luck with the visa application!

**Reply by JGTR** [reply]
They will be from this board, FYI: https://github.com/orgs/TelosLabs/projects/5

**Comment by LuigiR0jas** [comment]
Hey @yousamaa! Thanks for offering your help!

As @JGTR mentioned, we have a [handful of items](https://github.com/orgs/TelosLabs/projects/5) that we would like to include in our official launch for Rails World. You can find them in the "Ready" column and marked with the "Nice-to-have" label. Feel free to pick any issue that's not already assigned, and stay tuned as we will be adding new ones in the following days.

I suggest you take a look at our [Readme](https://github.com/TelosLabs/rails-world?tab=readme-ov-file#rails-world) file to learn how to run the app and familiarize yourself with our tooling and practices. Create a branch off of `main` when you feel ready to work on your contribution, and when you're done, just create a Pull Request so one of our maintainers can review it :)

Last but not least, you can also contribute by creating an issue to share a suggestion, improvement, bug report, or propose/request a new functionality you'd like to see in the app 🚀 

Feel free to ask any questions, and I hope we can meet in a couple of months!
